doc/docs/master/concepts/data-concepts/repo.md

```diff
@@ -37,25 +37,29 @@ Pachyderm's repositories are divided into two categories:
     User repositories keep track of your data one commit at a time. 
     They further split into:
 
-    - **Input** repositories
+    - **Source** repositories
```

Comment:
someone mentioned this renaming at some point, the idea being that even an "output" repo can be input to a pipeline, so a new word could be clearer

Reply:
Removed the term `Source`. Replaced with `Input repo with no provenance`.